### PR TITLE
add the money fox project

### PR DESCRIPTION
**Pull request by NPadrutt**

### CODE REVIEW

File: _data/projects/money-fox.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Money Manager*
```
Comment (shiftkey) [inline _data/projects/money-fox.yml]: Is this the right name? Does it need the asterisk?

Comment (NPadrutt) [inline _data/projects/money-fox.yml]: oh, no it isn't. Seems I didn't save right... Thanks for the hint.

File: _data/projects/money-fox.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+site: https://github.com/Apply-Solutions/MoneyManager
+tags:
+- ".net"
+- Xamarin
+- C#
+- mono
+- Android
+- UWP
+upforgrabs:
+  name: easy
```
Comment (shiftkey) [inline _data/projects/money-fox.yml]: Change this name  to `up-for-grabs` so that it matches up with the tag you use on the project...

Comment (NPadrutt) [inline _data/projects/money-fox.yml]: Pushed the change.
Thanks for the hint.. and sorry that it was necessary..

**Comment by shiftkey** [comment]
Thanks!

**Pull request merged**